quail/processes/wps_climdex_mmdmt.py

```diff
@@ -0,0 +1,148 @@
+import os
+from rpy2 import robjects
+from pywps import Process, LiteralInput
+from pywps.app.Common import Metadata
+from pywps.app.exceptions import ProcessError
+
+from wps_tools.utils import log_handler, collect_args, common_status_percentages
+from wps_tools.io import log_level
+from quail.utils import get_package, logger, load_rdata_to_python, save_python_to_rdata
+from quail.io import climdex_input, ci_name, output_file, rda_output, vector_name
+
+
+class ClimdexMMDMT(Process):
+    """
+    This process wraps climdex function
+    - climdex.txx: Monthly (or annual) Maximum of Daily Maximum Temperature
+    """
+
+    def __init__(self):
+        self.status_percentage_steps = dict(
+            common_status_percentages,
+            **{
+                "load_rdata": 10,
+                "save_rdata": 90,
+            },
+        )
+        inputs = [
+            climdex_input,
+            ci_name,
+            output_file,
+            LiteralInput(
+                "month_type",
+                "Month type to compute",
+                abstract="Min/ max daily temperature type to compute",
+                allowed_values=["txx"],
+                min_occurs=1,
+                max_occurs=1,
+                data_type="string",
+            ),
+            LiteralInput(
+                "freq",
+                "Frequency",
+                abstract="Time frequency to aggregate to",
+                allowed_values=["monthly", "annual"],
+                default="monthly",
+                min_occurs=0,
+                max_occurs=1,
+                data_type="string",
+            ),
```

Comment:
I saw a case that the input can take multiple values, like ` freq = c("monthly", "annual")`. How is this input dealing with that?